x-pack/plugins/ml/common/util/anomaly_utils.js

```diff
@@ -81,13 +104,21 @@ export function getSeverityColor(normalizedScore) {
 // which ranges from -5 to +5.
 export function getMultiBucketImpactLabel(multiBucketImpact) {
   if (multiBucketImpact >= MULTI_BUCKET_IMPACT.HIGH) {
-    return 'high';
+    return i18n.translate('xpack.ml.anomaliesTable.multiBucketImpact.highLabel', {
```

Comment:
Should these return types be changed to return objects with `id` and `label` props, to make them consistent with how the `getSeverity` functions have been changed? If so, the the `getMultiBucketImpactLabel` unit tests will need amending.

Reply:
I thought about this as well. The result of functions like getSeverity is used li css class name(in this case we need to have id property which is not localized) and like label on UI(in this case we need to have label property which is localized). But name of function `getMultiBucketImpactLabel` says that it returns only label, that is why I did not use object with `id` and `label` properties.

If in future we want to use result of `getMultiBucketImpactLabel` function also for css class names, it will be easy to convert result of `getMultiBucketImpactLabel` function from string to object with `id` and `label` properties.

So in my opinion it is a matter of taste :) @peteharverson I can make them consistent if in your opinion it would be better. Please let me know your decision.

Reply:
OK, makes sense not to change the return type of `getMultiBucketImpactLabel` as it isn't used currently for things like css class names. Thanks for the explanation.